verified_model/Site/North/reference_point_altitude.json

```diff
@@ -1,9 +1,9 @@
 {
-    "Value": 2157.0,
+    "value": 2158.0,
```

Comment:
This value should be 2156 I think. A validation step should definitely be included then.

Reply:
This is the position of the reference position, not corsika_obs_level. So it is actually wrong:

- reference_point_altitude is 2177 m (see [here](https://gitlab.cta-observatory.org/cta-science/simulations/simulation-and-performance-parallel-mode/layout-optimisation/layouts/-/blob/north-utm-updates/coordinates/telescope-list/NorthArray-utm.ecsv?ref_type=heads))
- corsika_obs_level is 2156 m in prod6 (was 2158 in prod5). 

Correct it (it was correct for the South).

Reply:
OK!